themes/src/main/resources/theme/base/login/template.ftl

```diff
@@ -34,6 +34,11 @@
             <script src="${script}" type="text/javascript"></script>
         </#list>
     </#if>
+    <#if authenticationSession??>
+        <script type="text/javascript">
```

Comment:
Ad mentioned in the comment above, if we convert this code to use modules we can do something like:

```html
<script type="module">
  import { checkCookiesAndSetTimer } from "/path/to/authChecker.js";
  
  checkCookiesAndSetTimer(
    "${authenticationSession.authSessionId}",
    "${authenticationSession.tabId}",
    "${url.ssoLoginInOtherTabsUrl}"
  );
</script>
```